users/src/main/java/com/ecommerce/users/services/JwtService.java

```diff
@@ -17,77 +17,59 @@
 
 @Service
```

Comment:
This file can be reverted. There are no logical diffs, just indentation changes.